lib/wibox/container/align.lua

```diff
@@ -0,0 +1,158 @@
+---------------------------------------------------------------------------
+-- A container used to align smaller widgets into larger space.
```

Comment:
This gives us a container called align and a layout called align, making the name "align" ambiguous. This needs another name (but I don't have any good suggestions, sorry).